frame/uniques/src/lib.rs

```diff
@@ -250,6 +258,11 @@ pub mod pallet {
 	pub(super) type CollectionMaxSupply<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Blake2_128Concat, T::CollectionId, u32, OptionQuery>;
 
+	#[pallet::storage]
+	/// Stores the last CollectionId of the collections.
```

Comment:
the count should always be +1 the last valid collection id.

when asset 0 is in storage, the count is 1

Reply:
Yeah, this is only a problem in the documentation.